slack_sdk/models/blocks/block_elements.py

```diff
@@ -1524,7 +1524,7 @@ def __init__(
                 Should be unique among all other action_ids in the containing block.
                 Maximum length for this field is 255 characters.
             options (required): An array of option objects to display in the menu.
-                Maximum number of options is 5, minimum is 2.
+                Maximum number of options is 5, minimum is 1.
```

Comment:
@seratch I ran both of the documentation steps and it resulted in a ton of changes. Happy to include those here, but wanted to be sure before I pushed them all up.

Reply:
@misscoded Thanks for checking that! More fixes would be appreciated 👍